cpp/dolfinx/mesh/Topology.cpp

```diff
@@ -269,5 +271,65 @@ std::string Topology::str(bool verbose) const
   return s.str();
 }
 //-----------------------------------------------------------------------------
+Eigen::Ref<const Eigen::Array<bool, 1, Eigen::Dynamic>>
+Topology::get_edge_reflections(const int cell_n) const
```

Comment:
It probably doesnt matter here, but cell indices are usually std::int32_t, right?